sources/packages/web/src/types/vuetify.ts

```diff
@@ -17,7 +15,7 @@ export interface ErrorMessage {
 /**
  * Vuetify v-form type definition.
  */
-export type VForm = Vue & {
```

Comment:
What is the reason for removing Vue. I believe it does not add any difference right?

Reply:
The object belonged to the removed package and did not cause any harm, hence it was removed.